CFile.js

```diff
@@ -42,69 +42,19 @@ var CFile = function(props) {
 };
 
 /**
- * Unescape the string to make the same string that would be
- * in memory in the target programming language.
- *
- * @static
- * @param {String} string the string to unescape
- * @returns {String} the unescaped string
- */
-CFile.unescapeString = function(string) {
-    var unescaped = string;
-
-    unescaped = unescaped.
-        replace(/\\\\n/g, "").                // line continuation
-        replace(/\\\n/g, "").                // line continuation
-        replace(/^\\\\/, "\\").             // unescape backslashes
-        replace(/([^\\])\\\\/g, "$1\\").
-        replace(/^\\'/, "'").               // unescape quotes
-        replace(/([^\\])\\'/g, "$1'").
-        replace(/^\\"/, '"').
-        replace(/([^\\])\\"/g, '$1"');
-
-    return unescaped;
-};
-
-/**
- * Clean the string to make a resource name string. This means
- * removing leading and trailing white space, compressing
- * whitespaces, and unescaping characters. This changes
- * the string from what it looks like in the source
- * code but increases matching.
- *
- * @static
- * @param {String} string the string to clean
- * @returns {String} the cleaned string
- */
-CFile.cleanString = function(string) {
-    var unescaped = CFile.unescapeString(string);
-
-    unescaped = unescaped.
-        replace(/\\[btnfr]/g, " ").
-        replace(/[ \n\t\r\f]+/g, " ").
-        trim();
-
-    return unescaped;
-};
-
-/**
- * Make a new key for the given string. This must correspond
- * exactly with the code in htglob jar file so that the
- * resources match up. See the class IResourceBundle in
- * this project under the java directory for the corresponding
- * code.
+ * Use a key for the given string as it is. Not manipulating at all.
  *
  * @private
  * @param {String} source the source string to make a resource
  * key for
  * @returns {String} a unique key for this string
  */
 CFile.prototype.makeKey = function(source) {
-    return CFile.unescapeString(source);
```

Comment:
I added the escaped function https://github.com/iLib-js/ilib-loctool-webos-c/blob/initialSetup/CFile.js#L52 so something like `\n or \t etc` is working. and I didn't wrap it when making a key. so I updated that part now. Regarding multi spaces in string, I'm not going to compress white space. so That's why only I added an `unescapedString` function.

Reply:
oh sorry.... ;(   I thought with `testCFileParseCSimple4` case,  it's ok. but I 've realized it doesn't work when I wrote  testcase in `t2.c` file.. It needs to handle something like `\n`\t` in strings. I'll re-update it..